dashboard/src/components/AppUpgrade/AppUpgrade.test.tsx

```diff
@@ -115,6 +115,41 @@ beforeEach(() => {
   spyOnUseHistory = jest
     .spyOn(ReactRouter, "useHistory")
     .mockReturnValue({ push: jest.fn() } as any);
+  // mock the window.matchMedia for selecting the theme
```

Comment:
We need to mock `window.matchMedia`, `window.ResizeObserver` and `HTMLCanvasElement.getContext()` as they are being used by monaco editor under the hood. The `jsdom` that `jest` uses does not include them, unfortunately.